years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```

Comment:
@vkuchyn why do we need to exclude `target`? Script should be invoked before target is created